.github/workflows/nightly-benchmark.yml

```diff
@@ -0,0 +1,22 @@
+name: Run nightly CI benchmarks
+
+on:
+  push:
+    branches:
+      - master
+      - 2.6
```

Comment:
Don’t we want a regex for version branches?

Reply:
indeed. addressed in the latest comment